app/controllers/auth/AuthAction.scala

```diff
@@ -130,7 +129,7 @@ class AuthActionImpl @Inject()(
         case _ => throw new RuntimeException("Can't find credentials for user")
       }
   } recover {
-    case _: NoActiveSession =>
+    case _: NoActiveSession | _: IncorrectCredentialStrength =>
```

Comment:
Having another look at this, I think all we want to do for this exception is: Redirect(configDecorator.authProviderChoice)